### PR TITLE
update proxy sha to cb503fe

### PR DESCRIPTION
**Pull request by yangminzhu**
To fix the test, this PR also update the pilot-agent inbound listener probe. The Envoy output at `127.0.0.1:15000/listeners` is changed by https://github.com/envoyproxy/envoy/pull/7131

cb503fe Update Envoy-WASM SHA to latest. (https://github.com/istio/proxy/pull/2295)
e2e9c43 Fix header parsing in JWT filter (https://github.com/istio/proxy/pull/2291)
716f81b Update Envoy WASM sha to the latest (https://github.com/istio/proxy/pull/2286)
6f1a58c Limit resource usage on Prow. (https://github.com/istio/proxy/pull/2289)
bfc559d Fix checks on master. (https://github.com/istio/proxy/pull/2287)
2a21f69 Set Istio authn filter to prefer using Envoy jwt filter if found (https://github.com/istio/proxy/pull/2281)
e954534 Update common files. (https://github.com/istio/proxy/pull/2280)
5c150dd Fix lint (https://github.com/istio/proxy/pull/2279)
b00c974 add insufficient include (https://github.com/istio/proxy/pull/2275)
af8f3c8 Report StopIteration if connection is closed (https://github.com/istio/proxy/pull/2270)
362fdf1 Update Envoy SHA to latest with option to select WASM runtimes. (https://github.com/istio/proxy/pull/2273)
59ad44d Add a simple setup for testing communication between 2 envoys (https://github.com/istio/proxy/pull/2262)
c77759c Use envoy-wasm as upstream (https://github.com/istio/proxy/pull/2252)
ac78dc0 Import common files into this repo. (https://github.com/istio/proxy/pull/2251)
5747f69 Replace qiwzhang who has left the project with crazyxy. (https://github.com/istio/proxy/pull/2241) (https://github.com/istio/proxy/pull/2243)
83f6566 Replace qiwzhang who has left the project with crazyxy. (https://github.com/istio/proxy/pull/2241)

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

For #9838

### CODE REVIEW

**Comment by yangminzhu** [comment]
We need to update the `go-control-plane` API to fix the remaining tests, blocked by https://github.com/envoyproxy/go-control-plane/issues/193
cc @kyessenov

File: pilot/cmd/pilot-agent/status/util/listeners.go [hunk 1]
```diff
@@ -31,28 +32,24 @@ var (
 
 // GetInboundListeningPorts returns a map of inbound ports for which Envoy has active listeners.
 func GetInboundListeningPorts(localHostAddr string, adminPort uint16, nodeType model.NodeType) (map[uint16]bool, string, error) {
-	buf, err := doHTTPGet(fmt.Sprintf("http://%s:%d/listeners", localHostAddr, adminPort))
+	buf, err := doHTTPGet(fmt.Sprintf("http://%s:%d/listeners?format=json", localHostAddr, adminPort))
```
Comment (howardjohn) [inline pilot/cmd/pilot-agent/status/util/listeners.go]: a bit annoying they changed the format on us, but doing this as json seems much better

**Pull request merged**